tests/storage/linstor/test_linstor_sr.py

```diff
@@ -157,10 +157,17 @@ def test_diskless_kept(self, host, linstor_sr, vm_on_linstor_sr):
         for member in host.pool.hosts:
             controller_option += f"{member.hostname_or_ip},"
 
+        # Determine the correct group name based on provisioning type
+        group_mapping = {
+            "thick": "xcp-sr-linstor_group_device",
+            "thin": "xcp-sr-linstor_group_thin_device"
```

Comment:
Use conftest.py here, we already have these helpers:
https://github.com/xcp-ng/xcp-ng-tests/blob/134fd551c4d2b5827b159da067020ddef79ac643/tests/storage/linstor/conftest.py#L9-L10

Reply:
Also `xcp-sr-linstor_group_device` is not valid, it's just `xcp-sr-linstor_group_device` for thick provisioning.

Reply:
Right, using `GROUP_NAME` is better.